packages/core/src/web/helpers/device-master.ts

```diff
@@ -1166,6 +1166,12 @@ class DeviceMaster {
     return controlSocket.addTask(controlSocket.rawMove, args);
   }
 
+  async rawWaitOkResponse(command: string, timeout?: number) {
+    const controlSocket = await this.getControl();
+
+    return controlSocket.addTask(() => controlSocket.useRawWaitOKResponse(command, timeout));
+  }
```

Comment:
Prefer `controlSocket.addTask(controlSocket.useRawWaitOKResponse, command, timeout);`
Because `control.ts` `doTask` use `taskFunction?.name` to track which function rejected